lib/commands/context.js

```diff
@@ -184,6 +204,14 @@ helpers.startChromedriverProxy = async function startChromedriverProxy (context)
       if (androidPackage && androidPackage.length > 0) {
         opts.chromeAndroidPackage = androidPackage[1];
       }
+      if (!opts.extractChromeAndroidPackageFromContextName) {
+        for (const wm of webviewsMapping) {
+          if (wm.webviewName === context && 'info' in wm && 'Android-Package' in wm.info) {
```

Comment:
this could be simplified to `_.has(wm?.info, 'Android-Package')`

Reply:
Addressed in a83c7b2